n2y/export.py

```diff
@@ -58,6 +59,14 @@ def export_page(
 ):
     page_properties = _page_properties(page, pandoc_format, id_property, url_property, property_map)
     pandoc_ast = page.to_pandoc()
+
+    if (n_empty_headers := _count_headerless_tables(pandoc_ast)) > 0:
```

Comment:
I propose that we split this into two lines to make it simpler

```
num_empty_headers = _count_headerless_tables(pandoc_ast)
if num_empty_headers > 0:
 ....
```

Reply:
I think the rest of the codebase uses "num_" instead of "n_". I'd see what's used elsewhere in the codebase and follow that.